### ISSUE TITLE
Error in Documentation on Page 146

### ISSUE THREAD

**Issue by dabiged** [OPM/opm-common#1485]
Page 146 of the pdf documentation of 2019-10 has the following code in the section "LIVEOIL":

> Example
> The following example activates the “black-oil” phases in the model.
> --
> -- ACTIVATE DEAD-OIL PHASE
> --
> LIVEDOIL 

I think the last line should read DEADOIL.

Apologies if this is the wrong place to submit this bug.

**Comment by OPMUSER** [comment]
No reporting the typo here is fine, nice to get some feedback for a change :-)

Page 146 is for the LIVEOIL keyword so it is the comment 

`-- ACTIVATE DEAD-OIL PHASE `

that should be changed to 

`-- ACTIVATE LIVE-OIL PHASE `

I have edited the manual already so the correction it will appear in the 2020-04 version.

Thanks for reporting.

**Comment by dabiged** [comment]
I guess the bug was more referring to the keyword "LIVEDOIL" which is invalid.

**Comment by OPMUSER** [comment]
No LIVEOIL is an E300 keyword, that is recognized by the parser but ignored by OPM Flow.

**Comment by bska** [comment]
> No LIVEOIL is an E300 keyword, that is recognized by the parser but ignored by OPM Flow.

True, of course, but in this case I think the user refers to an apparent misprint: "live**D**oil" with an extra "D" letter rather than just the expected "LIVEOIL" keyword.

**Comment by dabiged** [comment]
Yes that is what I was referring to.
LIVE**D**OIL is in the main part of that code and is not a valid keyword.

It should read:

```
Example
The following example activates the “black-oil” phases in the model.
--
-- ACTIVATE LIVE-OIL PHASE 
--
LIVEOIL 
```
NOT
```
Example
The following example activates the “black-oil” phases in the model.
--
-- ACTIVATE DEAD-OIL PHASE 
--
LIVEDOIL 
```
Note Change on the last line.

**Comment by OPMUSER** [comment]
As I  said I have already corrected this:

![image](https://user-images.githubusercontent.com/17958927/74797968-6dc93200-5307-11ea-8580-90a291856558.png)

But maybe my previous reply was not very clear and therefore adding to the confusion.
